samples/dnn/scene_text_detection.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * Real-time Scene Text Detection with Differentiable Binarization
+ * Copyright (C) 2020, The patent is owned by <Huazhong University of Science and Technology>
```

Comment:
Do I need to add some other information?
Pattent Name: 一种基于可微分二值化的实时文本检测方法
Publication Number: CN110781967A

Reply:
@vpisarev Please take a look on it.
Thank you for your suggestions in advance.

Reply:
@alalek I have talked with Vadim by email, and I have removed the patent information.

Reply:
If there is a reference/link on patented DNN model in this patch then it is still better to provide such type of information near the corresponding link (instead of whole sample).